drivers/serial/uart_nrfx_uarte.c

```diff
@@ -1462,7 +1474,16 @@ static int uarte_nrfx_pm_control(struct device *dev, u32_t ctrl_command,
 		irq_enable(DT_IRQN(UARTE(idx)));			       \
 	} while (0)
 
+#define HWFC_CONFIG_CHECK(idx) \
+	BUILD_ASSERT( \
+		(UARTE_PROP(idx, hw_flow_control) && \
+		(UARTE_HAS_PROP(idx, rts_pin) || UARTE_HAS_PROP(idx, cts_pin)))\
```

Comment:
You could use `UARTE_RTS_OR_CTS_PINS_SET(idx)` here, for better readability. This line should be also indented more.